### PR TITLE
ebuild-writing/users-and-groups: Update policy

### PR DESCRIPTION
**Pull request by mgorny**
Note: please don't push it until Council finished voting on GLEP 81 update.

### CODE REVIEW

**Comment by mgorny** [comment]
Reference policy: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Policies#GLEP_81_.28acct-.2A.29_package_policy

**Comment by mgorny** [comment]
Ping.

File: ebuild-writing/users-and-groups/text.xml [hunk 1]
```diff
@@ -6,21 +6,26 @@
...
-commit the new user and group packages.
+First, check the <uri link="https://api.gentoo.org/uid-gid.txt">UID/GID
+assignment list</uri> and choose a free UID/GID in range 101..499. If you are
+adding a matching user and group, use the same number for both. When in doubt,
+take the next free number from 499 downwards.
+</p>
+
+<p>
+The aforementioned list can be found in <uri
+link="https://gitweb.gentoo.org/data/api.git">data/api.git</uri> repository
```
Comment (ulm) [inline ebuild-writing/users-and-groups/text.xml]: s/can be found in/& the/

Don't split the `<uri>` tag between lines.

File: ebuild-writing/users-and-groups/text.xml [hunk 2]
```diff
@@ -6,21 +6,26 @@
...
+on <c>git.gentoo.org</c>. Add your new user(s) and group(s)
+to the <c>uid-gid.txt</c> file and push them before adding the actual packages.
+This counts as reserving the identifiers and prevents collisions. Afterwards,
```
Comment (ulm) [inline ebuild-writing/users-and-groups/text.xml]: s/prevents/will prevent/

**Comment by mgorny** [comment]
Both done.

**Pull request closed without merging**